easybuild/easyconfigs/g/GeneMark-ET/GeneMark-ET-4.72-GCCcore-12.3.0.eb

```diff
@@ -37,7 +37,11 @@ sanity_check_paths = {
 sanity_check_commands = [
     "gmes_petap.pl | grep 'Usage:'",
 ]
+skip_mod_files_sanity_check = True
 
 modextrapaths = {'PATH': ''}
+modextravars = {
+    'GENEMARK_PATH': '$EBROOTGENEMARKMINET',
```

Comment:
Seems like just %(installdir)s would be nicer here than this root envvar.

(or just do it like in https://github.com/easybuilders/easybuild-easyconfigs/pull/22661)